src/nvim/generators/gen_vimvim.lua

```diff
@@ -112,12 +112,13 @@ for _, au in ipairs(auevents.events) do
     w(' ' .. au)
   end
 end
-for au, _ in pairs(auevents.aliases) do
+for au, alias in pairs(auevents.aliases) do
   if not auevents.nvim_specific[au] then
```

Comment:
I think this condition should also be removed as it doesn't do anything, and the `au` variable above can be changed to `_`.

Reply:
done